src/_guides/language/evolution.md

```diff
@@ -269,10 +269,82 @@ except within the `@dart` and version strings.
 As the example above shows,
 other comments can appear before the `@dart` comment.
 
-
 For more information about how language versioning works, see the
 [language versioning specification][language versioning feature].
 
+
+## Mixed-mode programs
```

Comment:
I'm not a big fan of the term "mixed-mode". Null safety really isn't a mode, and I think the term makes the non-sound mode sound more appealing than it is.

My personal preference is to call it "partial null safety"; that makes it clear that you are not getting all soundness.

cc @leafpetersen

Reply:
We've decided on _unsound null safety_, per #2748.